src/radical/pilot/agent/resource_manager/base.py

```diff
@@ -216,17 +236,18 @@ def init_from_scratch(self):
         rm_info = RMInfo()
 
         # fill well defined default attributes
-        rm_info.requested_nodes  = self._cfg.nodes
-        rm_info.requested_cores  = self._cfg.cores
-        rm_info.requested_gpus   = self._cfg.gpus
-        rm_info.cores_per_node   = self._cfg.cores_per_node
-        rm_info.gpus_per_node    = self._cfg.gpus_per_node
-        rm_info.lfs_per_node     = self._cfg.lfs_size_per_node
-        rm_info.lfs_path         = ru.expand_env(self._cfg.lfs_path_per_node)
+        rm_info.requested_nodes       = self._cfg.nodes
+        rm_info.requested_cores       = self._cfg.cores
+        rm_info.requested_gpus        = self._cfg.gpus
+        rm_info.cores_per_node        = self._cfg.cores_per_node
+        rm_info.gpus_per_node         = self._cfg.gpus_per_node
+        rm_info.lfs_per_node          = self._cfg.lfs_size_per_node
+        rm_info.lfs_path              = ru.expand_env(self._cfg.lfs_path_per_node)
 
         rm_info.threads_per_gpu  = 1
         rm_info.mem_per_gpu      = None
-        rm_info.mem_per_node     = self._rcfg.mem_per_node or 0
+        rm_info.mem_per_node     = self._rcfg.mem_per_node    or 0
+        rm_info.numa_domain_map  = self._rcfg.numa_domain_map or {}
```

Comment:
we don't activate numa-domains yet, right? (i.e., Agent doesn't use this information)

Reply:
That is correct - but the upcoming scheduler changes will use it, so I wanted to have that readily available.